drakrun/drakrun/drakpdb.py

```diff
@@ -413,20 +415,21 @@ def make_pdb_profile(filepath, dll_origin_path=None, dll_path=None):
                 profile[target_key][next_sym_name] = mapped
 
     del mapped_syms
-    guid = pdb.STREAM_PDB.GUID
-    guid_str = "%.8X%.4X%.4X%s" % (
-        guid.Data1,
-        guid.Data2,
-        guid.Data3,
-        guid.Data4.hex().upper(),
+    pdb_guid = pdb.STREAM_PDB.GUID
+    pdb_guid_str = "%08x%04x%04x%s" % (
+        pdb_guid.Data1,
+        pdb_guid.Data2,
+        pdb_guid.Data3,
+        pdb_guid.Data4.hex(),
     )
```

Comment:
I think we could extract it into a separate function

Reply:
I planned refactoring on the next PR.